### PR TITLE
Check for the spec version when determining private read receipt support

### PR DESCRIPTION
**Pull request by SimonBrandner**
---
This change is marked as an *internal change* (Task), so will not be included in the changelog.

### CODE REVIEW

File: src/components/structures/TimelinePanel.tsx [hunk 1]
```diff
@@ -994,7 +994,8 @@ class TimelinePanel extends React.Component<IProps, IState> {
                     if (e.errcode === 'M_UNRECOGNIZED' && lastReadEvent) {
                         if (
                             !sendRRs
-                            && !cli.doesServerSupportUnstableFeature("org.matrix.msc2285.stable")
+                            && !await cli.doesServerSupportUnstableFeature("org.matrix.msc2285.stable")
+                            && !await cli.isVersionSupported("v1.4")
```
Comment (turt2live) [inline src/components/structures/TimelinePanel.tsx:998]:
```suggestion
                            && !(await cli.doesServerSupportUnstableFeature("org.matrix.msc2285.stable"))
                            && !(await cli.isVersionSupported("v1.4"))
```

I'm paranoid about the runtime doing the wrong thing, sorry

File: src/components/views/settings/tabs/user/PreferencesUserSettingsTab.tsx [hunk 2]
```diff
@@ -125,9 +125,12 @@ export default class PreferencesUserSettingsTab extends React.Component<IProps,
     }
 
     public async componentDidMount(): Promise<void> {
+        const cli = MatrixClientPeg.get();
+
         this.setState({
             disablingReadReceiptsSupported: (
-                await MatrixClientPeg.get().doesServerSupportUnstableFeature("org.matrix.msc2285.stable")
+                await cli.doesServerSupportUnstableFeature("org.matrix.msc2285.stable")
+                || await cli.isVersionSupported("v1.4")
```
Comment (turt2live) [inline src/components/views/settings/tabs/user/PreferencesUserSettingsTab.tsx:133]:
```suggestion
                (await cli.doesServerSupportUnstableFeature("org.matrix.msc2285.stable"))
                || (await cli.isVersionSupported("v1.4"))
```

**Review by turt2live — APPROVED** [review]
otherwise lgtm

File: src/components/structures/TimelinePanel.tsx (hunk 1 above)
Comment (SimonBrandner) [reply src/components/structures/TimelinePanel.tsx:998]: Understandable

**Pull request merged**